docs/intro/quickstart.rst

```diff
@@ -35,6 +35,13 @@ CLI on your machine. You may be asked for your password. Once the installation
 completes, you may need to **restart your terminal** before you can use the
 ``edgedb`` command.
 
+.. note::
+
+  Check out our additional installation methods on `Linux\
+  </install#linux-debianubuntults>`_, `macOS\
+  </install#macos-homebrew>`_, and `Windows\
+  </install#windows-commandprompt>`_.
+
```

Comment:
Maybe still mention what a few of the methods are, so it's clear that they include windows command prompt and homebrew.

Reply:
I think the changes just pushed will address this. Let me know if you see anything else that needs attention!